README.md

```diff
@@ -346,6 +346,7 @@ POSTGRES_PASSWORD=$(kubectl get secret artifactory-ha-postgresql -o jsonpath="{.
 ```bash
 helm upgrade --install artifactory-ha  jfrog/artifactory-ha \
     --set artifactory.joinKey=$JOIN_KEY \
+    --set artifactory.masterKey=$MASTER_KEY \
```

Comment:
There's no need to provide MASTER_KEY when upgrading artifactory. JOIN_KEY should be enough.